roles/confluent.zookeeper/tasks/main.yml

```
@@ -14,6 +14,21 @@
     - ansible_os_family
     - ansible_fqdn
 
+- name: Get Package Facts
+  package_facts:
+    manager: auto
+  when: installation_method == "package"
+
+- name: Stop Zookeeper Service if upgrade
+  ansible.builtin.systemd:
```

is there a benefit to moving to `ansible.builtin.systemd` over `systemd`? We use `systemd` elsewhere so its a bit of a break in pattern.

I have noticed the docs getting updated to using ansible.builtin, but its not clear if its necessary.

---

```suggestion
  systemd:
```
No, we are using it. So I forgot to fix it.